contrib/ftpasswd

```diff
@@ -808,6 +808,15 @@ sub handle_group_entry {
     } else {
       $members = $add_user;
     }
+
+    # Remove any duplicates and resort
```

Comment:
Is this solving a functional problem, or is this more for esthetics?

Reply:
1. The sort:  is prob. more esthetics than functional but makes it easier to read.  
2. The dups:  For us it is just easier to audits if there are no duplicates and users are less prone to make mistakes as the list gets long.
3. the documentation: functional

Reply:
Actually, the PR fixed another issue.  If dups are allowed, the --delete-member left a trailing comma.   See below 


```
ian:~/git-repo/proftpd/contrib$ ./ftpasswd --group --name foo --gid 101 --add-member foo --member bar --member foo --member foo
ftpasswd2: creating group entry for group foo
ftpasswd2: entry created

ian:~/git-repo/proftpd/contrib$ cat ftpd.group
foo:x:101:bar,foo,foo,foo

ian:~/git-repo/proftpd/contrib$ ./ftpasswd --group --name foo --gid 101 --delete-member foo
ftpasswd2: updating group entry for group foo
ftpasswd2: entry updated

ian:~/git-repo/proftpd/contrib$ cat ftpd.group
foo:x:101:bar,
ian:~/git-repo/proftpd/contrib$
```